### DISCUSSION TITLE
Can we has semantic versioning ?

### DISCUSSION THREAD

**Discussion by utunga** [CredComSoc/ProtocolWorkingGroup#14] [Reference Implementation]
It would be helpful if we had clear semantic versioning of the ***current*** code base.  Given that other projects are now integrating / dependent on cc-server/cc-node this has become more important.

That said this doesn't need to be that hard. We don't need to overthink this...

Step 1:  Choose a number to start with.
Step 2:  Add a tag for current HEAD with that number in gitlab (in all/both repos).  

Step 2 is purely mechancial. Step 1 requires a bit of a judgement call and ultimately would be up to @matslats .

My 2c on the subject is I'm be OK with whatever number Mats chooses, but. thinking it might be useful for me to offer an opinion I'd say something in the general vicinity of 1.x.x seems about right. Probably less than 1.0.0 to signify that we're not quite fully ready for 'version 1' status. On the other hand something at or higher than 1.0.0 if we want to imply that it is already in production. At the end of the day the most important thing is to have *a* tag so that we can easily indicate what other software has been tested against and will integrate with smoothly etc.

**Comment by matslats** [comment]
Ok Following Miles request I'm happy to go with version 0.8, for all of
The protocol (most important)
cc-php-lib
cc-node
cc-server
to imply that we are well on the way to version 1 of the API.
At the moment all they are all in lockstep which reflects that they are all maintained simultaneously by me.
I would suggest a meeting to produce a road map, where we plan anticipated version numbers and the nearer new features.
I will make this tag today if I don't hear suggestions to the contrary.

**Comment by utunga** [comment]
Brilliant thanks so much. The tag would be appreciated. 

And as for the conversation, yes let's do that, happy to do so, perhaps as part of wider discussion which we will hopefully get back to soon.
